pkg/csi_driver/multishare_controller.go

```diff
@@ -71,16 +84,37 @@ func NewMultishareController(config *controllerServerConfig) *MultishareControll
 		ecfsDescription: config.ecfsDescription,
 		isRegional:      config.isRegional,
 		clustername:     config.clusterName,
+		kubeClient:      config.kubeClient,
 	}
 	c.opsManager = NewMultishareOpsManager(config.cloud, c)
 	if config.features != nil && config.features.FeatureMaxSharesPerInstance != nil {
 		c.featureMaxSharePerInstance = config.features.FeatureMaxSharesPerInstance.Enabled
 		c.descOverrideMaxSharesPerInstance = config.features.FeatureMaxSharesPerInstance.DescOverrideMaxSharesPerInstance
 		c.descOverrideMinShareSizeBytes = config.features.FeatureMaxSharesPerInstance.DescOverrideMinShareSizeGB
+
+		// The PV informer is needed to determine the share capacity ranges, irrespective of whether the feature is enabled or not.
```

Comment:
IIUC the pv informer is only used `if m.featureMaxSharePerInstance `?

Reply:
This is a stale comment. Fixed.

Please also check the feature disable behavior [here](https://docs.google.com/document/d/1Jsx2H90NPjP8kddrljwwi0LKIlAmkT_ImAa_f4mwUfI/edit#bookmark=id.nqfsjuj8upfu)